packages/serp-report/src/content_scripts/serp-report.js

```diff
@@ -168,6 +177,13 @@ function renderWheel(WTMTrackerWheel, anchor, stats) {
       return;
     }
 
+    if (message.data === 'WTMReportDisable') {
+      closePopups();
+      elements.forEach(removeWheel);
+      chrome.runtime.sendMessage({ action: 'disableWTMReport' });
```

Comment:
Would be good to define actions as constants and reuse in all contexts but the fact that content script don't support JavaScript modules makes it awkward.

Reply:
I thought about it, but as it requires to use dynamic import, I didn't want to make to complex code here.